config/default/shifts.json

```diff
@@ -0,0 +1,74 @@
+{
+    "R" : {
+        "code" : "R",
+        "from" : 7,
+        "to" : 15,
+        "color" : "pink",
```

Comment:
Nie musisz tutaj umieszczać koloru, nawet jeśli będziemy to dostawać z frontendu, to nie znaczy, że musimy używać.

Reply:
To samo tyczy się name

Reply:
Prawda, po prostu konwencję ze _schedule_ pisząc je. Mogę wywalić to jak chcesz

Reply:
Wywal, bo oni sobie zrobią resztę jak chcą, my tylko to narzucamy, bo się pogubią :P